### ISSUE TITLE
Feature request: extend `%header.set()%` to accept multiple addressees

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplate4#327]
To simplify usage, the commands `header.set(to)`, `header.set(cc)` , `header.set(bcc)` could be expanded to accept multiple addressee parameters, like this:

```
%header.set(to,"address1","address2","address3")%
```

The address parameters can be the usual 
`name <email address> `
or
`email address`

Currently you can only achieve multiple recipients by using a combination of commands like this:

```
%header.set(to,"address1")%
%header.append(to,"address2")%
%header.append(to,"address3")%
```

It will also make sense to allow multiple parameters to `%header.append()%` - in order to keep the original recipient in a reply template and add multiple recipients in a single line.

**Comment by RealRaven2000** [comment]
Test version below:

[smartTemplate-fx-4.9pre77.zip](https://github.com/user-attachments/files/17396607/smartTemplate-fx-4.9pre77.zip)

----

To install version above download zip file and drag the file into Thunderbird Add-ons Manager (do not extract contents, it won't install like that)

**Comment by JohnFLand** [comment]
This new feature worked perfectly for me! Thank you for taking the time to address my issue referencing multiple addressees!

**Comment by RealRaven2000** [comment]
Implemented in 4.9 - published 11/11/2024
